releases/R2023b/README.md

```diff
@@ -0,0 +1,263 @@
+    | **IP Addresses Allowed to Access MATLAB Web App Server Apps Home Page** | Complete this field only if you selected **"No"** in the previous field. Specify the range of IP addresses that can access the MATLAB Web App Server apps home page in CIDR notation. The format for this field is IP Address/Mask.<p><em>*Example*</em>: `x.x.x.x/24`</p> |
+    | **ARN of SSL Certificate** | Specify the Amazon Resource Name (ARN) of the SSL certificate you uploaded to the AWS Ceritifcate Manager. The ARN facilitates connecting to the apps home page using an HTTPS connection.<p><em>*Example*</em>: <code>arn:aws:acm:us-east-1:012345678910:certificate/666abcd6-ab6c-6ab6-a666-a666666bcd66</code> </p><p>To retrieve an ARN:</p><ul><li>Type "Certificate Manager" in the search box at the top of the web page and hit Enter. This automatically takes you to the AWS Certificate Manager.</li><li>Select the certificate you uploaded based on Certificate ID, or Domain name, or Name tag .</li><li>Copy the ARN from the "Certificate status" section at the top.</li></ul><p>For more information, see [Create Self-signed Certificate](/README.md#create-self-signed-certificate) and [Upload Self-signed Certificate to AWS Certificate Manager](/README.md#upload-self-signed-certificate-to-aws-certificate-manager).
+    | **EC2 Instance Type** | Choose the AWS EC2 instance type to use for the server. All AWS instance types are supported. For more information, see [Amazon EC2 Instance Types](https://aws.amazon.com/ec2/instance-types/). <p><em>*Example*</em>: `m5.xlarge`</p> |
+    | **Operating System** | Choose between Windows (Windows Server) and Linux (Ubuntu).  |
+    ||**Settings for Network License Manager**|
+    | **Password for Network License Manager** | Specify a password for the network license manager. Use this password to log in to the network license manager after the stack has been successfully created.<p>Deploying MATLAB Web App Server automatically deploys a network license manager.</p>|
+    | **Confirm Password** | Reenter the password to log in to the network license manager. |
+    ||**If you are using [an existing VPC](#How-do-I-use-an-existing-VPC-to-deploy-MATLAB-Web-App-Server?), the following settings are available:**|
+    | **Port and IP Address of Existing Network License Manager** | Enter the IP address and port number of the existing network license manager. You can find the private IP address in the *Outputs* tab of the existing network license manager deployment.  |
```

Comment:
I feel this parameter and the following one need to go in the existing VPC section where we mention all other parameters for the existing VPC template.

For this parameter, it would be better if we mentioned port number followed by the private IP address. Would also help if we gave an example. MPS has the same field. You could refer to the language there and reuse it here if you see fit: https://github.com/mathworks-ref-arch/matlab-production-server-on-aws/blob/master/releases/R2023b/README.md.
Also mention that it is an optional parameter.